app/controllers/items_controller.rb

```diff
@@ -1,20 +1,21 @@
 class ItemsController < ApplicationController
-  def home
-    @item = Item.new
-    @items = Item.all
-  end
 
   def create
-    @item = Item.new(item_params)
+    @inventory_dashboard = InventoryDashboard.new(Item.all)
 
-    if @item.save
-      redirect_to root_path
+    if @inventory_dashboard.save_item(item_params)
+      flash[:error] = 'error message'
+      render :index
     else
-      flash[:error] = "Invalid"
+      flash[:success] = 'success'
       redirect_to root_path
     end
   end
 
+  def index
+    @inventory_dashboard = InventoryDashboard.new(Item.all)
+  end
+
```

Comment:
@Ru-T try `@inventory_dashboard.items`. The instance variables and the methods inside of the dashboard object aren't directly accessible, but they can be accessed through the `@inventory_dashboard` object.